src/functions/uploadToRSIS/application/__mocks__/result-client.mock.ts

```diff
@@ -0,0 +1,86 @@
+// In order to use this mock just swap the imports in the batch-processor.ts
+// and inport everything from here.
+
+// This code has some mock data for test results so that the lambda does not have to fetch
+// the test results externally
+
+import { info } from '@dvsa/mes-microservice-common/application/utils/logger';
+import { StandardCarTestCATBSchema, ApplicationReference } from '@dvsa/mes-test-schema/categories/B';
+
+import { pass, failAllFaults, failAllSerious, failAllDangerous, rekeyPass } from './result-uploads.mock';
+
+// Needs to kept in sync with the PROCESSING_STATUS table
+export enum ProcessingStatus {
+  PROCESSING = 'PROCESSING',
+  ACCEPTED = 'ACCEPTED',
+  FAILED = 'FAILED',
+}
```

Comment:
Can the mock depend on the real result-client.ts and import rather than repeat the types, so we don't duplicate the code?

Reply:
Sure, deleted these ones and importing the real ones from `result-client.ts`